server/controllers/stock/core.js

```diff
@@ -272,7 +272,7 @@ function getLotsMovements(depotUuid, params) {
       m.flux_id, BUID(m.entity_uuid) AS entity_uuid, m.unit_cost,
       f.label AS flux_label, i.delay,
       iu.text AS unit_type,
-      dm.text AS documentReference, m.period_id
```

Comment:
I think @lomamech added these `period_id`s in.  Did you mean to remove them?  Is there something they screw up?